lib/init/grass.py

```diff
@@ -1584,7 +1585,6 @@ def close_gui():
     env = gcore.gisenv()
     if 'GUI_PID' not in env:
         return
-    import signal
```

Comment:
Perhaps minor speedup, but this is currently needed only for the GUI and in general we should minimize the overhead when grass.py is called in command line to do batch jobs. (`import types` is similar case, although not related to GUI vs batch) Just a note. I'm leaving it up to you.